frontend/src/components/MapView/LeftPanel/layersPanel/MenuSwitch/SwitchItem/LayerDownloadOptions.tsx

```diff
@@ -57,7 +58,9 @@ function LayerDownloadOptions({
   const { startDate: selectedDate } = useSelector(dateRangeSelector);
   const serverAvailableDates = useSelector(availableDatesSelector);
   const layerAvailableDates = serverAvailableDates[layer.id];
-  const queryDate = getRequestDate(layerAvailableDates, selectedDate);
+  const queryDate = selected
```

Comment:
I would suggest adding typing annotations to the arguments of `LayerDownloadOptions`, but otherwise, looks good to me.

Reply:
You mean LayerDownloadOptionsProps ?

Reply:
pff, yes. I still get caught by TS syntax :( Ignore me, sorry.